src/zenml/integrations/kubernetes/orchestrators/kubernetes_orchestrator.py

```diff
@@ -576,15 +579,19 @@ def submit_pipeline(
                 labels=orchestrator_pod_labels,
             )
 
-            self._k8s_batch_api.create_namespaced_cron_job(
+            cron_job = self._k8s_batch_api.create_namespaced_cron_job(
                 body=cron_job_manifest,
                 namespace=self.config.kubernetes_namespace,
             )
             logger.info(
                 f"Scheduling Kubernetes run `{pod_name}` with CRON expression "
                 f'`"{cron_expression}"`.'
             )
-            return None
+            return SubmissionResult(
+                metadata={
+                    KUBERNETES_CRON_JOB_METADATA_KEY: cron_job.metadata.name,
+                }
+            )
```

Comment:
Just checking: this means that users with existing schedules won't be able to delete them or update them, correct ?